lexos/helpers/constants.py

```diff
@@ -161,6 +161,11 @@
     'tolerance',
     'KMeans_metric',
     'viz')
+# for kmeans analyzing
+N_INIT = 300
+MAX_ITER = 300
+TOLERANCE = 1e-4
```

Comment:
can we have some documentation on these three values?  e.g.,  TOLERANCE (of what?)